Modules/Loadable/VolumeRendering/MRML/vtkMRMLVolumePropertyNode.cxx

```diff
@@ -646,9 +637,31 @@ std::string vtkMRMLVolumePropertyNode::GetRGBTransferFunctionAsString()
 //---------------------------------------------------------------------------
 void vtkMRMLVolumePropertyNode::SetRGBTransferFunctionAsString(std::string rgbTransferFunctionStr)
 {
-  vtkColorTransferFunction* colorTransfer = vtkColorTransferFunction::New();
+  vtkNew<vtkColorTransferFunction> colorTransfer;
   this->GetColorTransferFunctionFromString(rgbTransferFunctionStr.c_str(), colorTransfer);
   this->SetColor(colorTransfer);
-  colorTransfer->Delete();
-  this->Modified();
+}
+
+//---------------------------------------------------------------------------
+void vtkMRMLVolumePropertyNode::SetPropertyInVolumeNode(vtkVolume* volume)
+{
+  if (!volume)
+  {
+    vtkGenericWarningMacro("SetPropertyInVolumeNodeWithoutModifications: Invalid volume");
+    return;
+  }
+  vtkVolumeProperty* volumeProperty = this->GetVolumeProperty();
+
+  // vtkVolume::SetProperty(vtkVolumeProperty*) calls Modified() on the property.
+  // We need to termporarily ignore Modified events.
```

Comment:
Minor typo being warned by codespell in your last commit in this PR.
```suggestion
  // We need to temporarily ignore Modified events.
```

Reply:
Thank you, fixed.